frontend/src/scenes/instance/AsyncMigrations/index.tsx

```diff
@@ -200,11 +238,17 @@ export function AsyncMigrations(): JSX.Element {
                                 </Button>
                             </div>
                             <Space />
-                            <Table
-                                pagination={false}
+                            <LemonTable
+                                pagination={{ pageSize: 10 }}
                                 loading={asyncMigrationsLoading}
                                 columns={columns}
                                 dataSource={asyncMigrations}
+                                expandable={{
+                                    expandedRowRender: function renderExpand(row) {
+                                        return row && <AsyncMigrationDetails asyncMigration={row} />
+                                    },
+                                    rowExpandable: ({ error_cnt }) => error_cnt > 0,
+                                }}
```

Comment:
why does the rendering look weird, when it shouldn't be expandable?

Reply:
Sorry, this is a bug I introduced! Fixed now on this branch

Reply:
The [coding convention dos & donts here](https://posthog.com/docs/contribute/coding-conventions#do-s--dont-s) suggest not using "index.tsx" as a filename.